### ISSUE TITLE
frint-react: Empty Component when Region is empty

### ISSUE THREAD

**Issue by fahad19** [Travix-International/frint#216]
## Current situation

Right now, when a Region has no Apps to render, it just remains empty.

It would be useful if we can pass some default Component to render in those cases.

## Proposed API

```js
import EmptyRegionMessage from '../components/EmptyRegionMessage';

<Region name="sidebar" empty={EmptyRegionMessage} />
```

If the sidebar region is empty, it would render `EmptyRegionMessage` component by default.

**Comment by discosultan** [comment]
Nice idea!

As for naming, using something like `default` sounds more intuitive in my opinion. Mainly because the keyword is common in different languages (SQL, js, C#, etc...) for expressing a similar concept.

Should it also enable specifying a collection of default components?

**Comment by fahad19** [comment]
`default` is tricky, just like `class`, `module` and `exports` as variable names in JavaScript.

**Comment by asci** [comment]
Yep, awesome idea. I like `empty` because it is the same as we do for lists.
